conf.py

```diff
@@ -67,7 +67,9 @@
 ])
 
 source_constants = {
-    'odbc-driver': '`MongoDB ODBC Driver for BI Connector <https://github.com/mongodb/mongo-odbc-driver/releases/tag/v1.0.0>`_'
+    'odbc-driver': '`MongoDB ODBC Driver for BI Connector <https://github.com/mongodb/mongo-odbc-driver/releases/tag/v1.0.0>`_',
+    'atlas-bi-short': 'BI Connector for Atlas',
```

Comment:
This is pretty nitty, but can we change to `bi-atlas-short` so it matches the pipe one above? Then find/replace `{+atlas-bi-short+}` with `{+bi-atlas-short+}`

Reply:
Done, and here's the staged link: https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/steverenaker/DOCSP-1881/index.html